src/Build/Graph/ProjectGraphNode.cs

```diff
@@ -3,13 +3,15 @@
 
 using System.Collections.Generic;
 using System.Diagnostics;
+using Microsoft.Build.BackEnd;
 using Microsoft.Build.Execution;
 using Microsoft.Build.Shared;
 
 namespace Microsoft.Build.Experimental.Graph
 {
     /// <summary>
     /// Represents the node for a particular project in a project graph.
+    /// A node is defined by (ProjectPath, ToolsVersion, GlobalProperties).
```

Comment:
I don't think `ToolsVersion` is actually part of it. Unless things changed from the initial design.

Reply:
[They do contribute to the project's unique ID](https://github.com/cdmihai/msbuild/blob/2c7610b154bce223bfefadd147109aac2c631160/src/Build/BackEnd/Shared/ConfigurationMetadata.cs#L165-L168), but now that we're practically ignoring ToolsVersion all over the place it's not that important anymore.